agent/config/config.go

```diff
@@ -312,6 +312,8 @@ type Config struct {
 	SyncCoordinateRateTarget   *float64 `json:"sync_coordinate_rate_target,omitempty" hcl:"sync_coordinate_rate_target" mapstructure:"sync_coordinate_rate_target"`
 	Version                    *string  `json:"version,omitempty" hcl:"version" mapstructure:"version"`
 	VersionPrerelease          *string  `json:"version_prerelease,omitempty" hcl:"version_prerelease" mapstructure:"version_prerelease"`
+
+	EnqueueLimit *string `json:"enqueue_limit,omitempty" hcl:"enqueue_limit" mapstructure:"enqueue_limit"`
```

Comment:
Please move that up a little, it ended up under `// non-user configurable values` which is not correct. Below `Watches` seems like a good place, would be similar to the position in `builder.go`.

Reply:
Thanks, I moved the place in the struct.